apisix/plugins/example-plugin.lua

```diff
@@ -85,6 +90,10 @@ function _M.access(conf, ctx)
     core.log.warn("plugin access phase, conf: ", core.json.encode(conf))
     -- return 200, {message = "hit example plugin"}
 
+    if conf.lua_proxy_upstream then
```

Comment:
This is a feature PR that has added plugin configuration and requires updating schema and documentation.